stingray/io.py

```diff
@@ -315,37 +325,74 @@ def _save_pickle_object(object, filename, **kwargs):
             pickle.dump(object, f)
 
 def _retrieve_pickle_object(filename):
+    """
+    Retrieves a pickled class object.
+
+    Parameters:
+    -----------
```

Comment:
If I recall correctly, in doc strings, colon `:` is not required if the key word is underlined.